### PR TITLE
fourth commit from Steve Jeong

### PR DESCRIPTION
**Pull request by Steve-Jeong**
please accept my fourth commit

### CODE REVIEW

**Comment by Steve-Jeong-Galaxy** [comment]
what is this for?

**Comment by Steve-Jeong** [comment]
to make it fourth commit

File: mywork.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 first
 second
-third
\ No newline at end of file
+third
+fourth
```
Comment (Steve-Jeong-Galaxy) [inline mywork.txt]:
```suggestion
4th
```

File: fourth.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+fourth.txt
```
Comment (Steve-Jeong-Galaxy) [inline fourth.txt]:
```suggestion
fourth.txt (4th)
```

**Review by Steve-Jeong-Galaxy — COMMENTED** [review]
Please check my comments

**Comment by Steve-Jeong-Galaxy** [comment]
good job. anything else?

**Comment by Steve-Jeong** [comment]
no more

**Pull request merged**